framework/src/modules/reward/module.ts

```diff
@@ -102,7 +102,7 @@ export class RewardModule extends BaseModule {
 		await this._tokenAPI.mint(
 			context.getAPIContext(),
 			context.header.generatorAddress,
-			this._tokenIDReward,
+			this._tokenID,
```

Comment:
I think the variable name was aligned with other modules

Reply:
After discussion will keep as is